pages/linux/raspinfo.md

```diff
@@ -0,0 +1,8 @@
+# raspinfo
+
+> Display Raspberry Pi system information.
+> More information: <https://github.com/raspberrypi/utils/tree/master/raspinfo>.
+
+- Display Raspberry Pi system information:
```

Comment:
Do we need to repeat the platform here?
```suggestion
- Display system information:
```

Reply:
Not particularly, however its just a little extra info if the person ignored the duplicate command description

Reply:
I don't have a strong opinion about it. If we keep it I would suggest `Display system information about a Raspberry Pi` though.

Reply:
That kinds of makes it sound like a manual for raspberry pis, after looking at this again, I like Axel's wording of it since it more directly correlates to it being the system you are currently on.